aesd-char-driver/main (copy).c

```diff
@@ -0,0 +1,267 @@
+/**
```

Comment:
@MuktaDarekar don't make copies of files in your git repo, let git manage revision history for you.